phaseshifts/wrappers.py

```
@@ -0,0 +1,752 @@
+            slab_mtz = Converter.import_CLEED(slab_file)
+            full_fname = glob(os.path.expanduser(os.path.expandvars(slab_file)))[0]
+            slab_file = os.path.join(
+                tmp_dir, os.path.splitext(os.path.basename(full_fname))[0] + "_slab.i"
+            )
+            slab_mtz.gen_input(filename=slab_file)
+        else:
+            slab_mtz.load_from_file(slab_file)
+
+        # generate atomic charge densities for each element in bulk model
+        if not isinstance(bulk_mtz, model.MTZ_model):
+            raise AttributeError("bulk_mtz is not an MTZ_model() instance")
```

Ruff (0.14.10)

Prefer `TypeError` exception for invalid type

(TRY004)